### PR TITLE
Github-34518: fixes 2 broken links

### PR DESCRIPTION
**Pull request by JoeAldinger**
4.6+ 

Github issue #34518

[Preview](https://deploy-preview-41655--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/managing_images/using-image-pull-secrets.html)

Fixes broken links that returned 404 error

### CODE REVIEW

File: openshift_images/managing_images/using-image-pull-secrets.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ However, for other scenarios, such as referencing images across {product-title}
 
 You can obtain the image pull secret, `pullSecret`, from the link:https://console.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {console-redhat-com} site.
 
-You use this pull secret to authenticate with the services that are provided by the included authorities, including link:quay.io[Quay.io] and link:registry.redhat.io[registry.redhat.io], which serve the container images for {product-title} components.
+You use this pull secret to authenticate with the services that are provided by the included authorities, from the link:https://quay.io/[Quay.io] and from the link:https://registry.redhat.io[registry.redhat.io], which serve the container images for {product-title} components.
```
Comment (stevsmit) [inline openshift_images/managing_images/using-image-pull-secrets.adoc:15]: The links work fine but I'm not crazy about 'from the.' I think it reads better as follows: 

You use this pull secret to authenticate with the services that are provided by the included authorities from link:https://quay.io/[Quay.io] and link:https://registry.redhat.io[registry.redhat.io], which serve the container images for {product-title} components.

Comment (JoeAldinger) [reply openshift_images/managing_images/using-image-pull-secrets.adoc]: Removed "from the".

**Pull request merged**